uaa/add-user.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+if [ "$#" -lt 1 ]; then
+  echo
+  echo "Usage:"
+  echo "  $ ./add-user.sh <EMAIL_ADDRESS>"
+  echo
+  exit 1;
+fi
+
+if ! hash uaac 2>/dev/null; then
+  echo -n "uaac must be installed"
+  exit 1
+fi
+
+EMAIL=$1
+ORIGIN=$(awk -F '@' '{print $2}' <<< "$EMAIL")
```

Comment:
Is it too clever to determine the origin from the email? it seemed safe to me and worked in testing

Reply:
Hm, maybe just note that behavior in the usage text? I can see someone wanting to create a user from the cloud.gov origin with an agency email.